### DISCUSSION TITLE
How to include non-paid channels in Meridian

### DISCUSSION THREAD

**Discussion by yongqiangzhangzazzle** [google/meridian#139] [Q&A] [answered]
We have non-paid traffic such as SEO and direct. Should we include them into "controls" when building the MMM model with Meridian?

**Comment by jyklly** [comment, marked as answer]
We are exploring the possibility of adding support for non-paid channels to Meridian. In the meantime, including them as controls is a good idea if they are likely confounders for your use case. [https://developers.google.com/meridian/docs/advanced-modeling/control-variables](https://developers.google.com/meridian/docs/advanced-modeling/control-variables) contains more information on confounders and selecting control variables.
